amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/fanout/FanOutConsumerRegistrationTest.java

```diff
@@ -90,8 +90,8 @@ public void testConsumerAlreadyExists() throws Exception {
 
         assertThat(consumerArn, equalTo(CONSUMER_ARN));
 
-        verify(client).describeStreamConsumer(eq(createDescribeStreamConsumerRequest(null)));
-        verify(client).describeStreamSummary(eq(createDescribeStreamSummaryRequest()));
+        verify(client, times(2)).describeStreamConsumer(any(DescribeStreamConsumerRequest.class));
+        verify(client).describeStreamSummary(any(DescribeStreamSummaryRequest.class));
```

Comment:
This falls under "Test behavior, not implementation" 

Does it really matter that we are calling describeStreamConsumer, or is verifying that we never register the already existing consumer again really the behavior we are looking to confirm?

Reply:
Yep, let's remove these describeStreamConsumer and describeStreamSummary verification